src/main/java/com/thullo/service/TaskServiceImpl.java

```diff
@@ -98,8 +99,7 @@ public Task moveTask(String boardRef, String status, Long position) throws Resou
             }
             task.setPosition(index);
         }
-
-        task.setStatus(status);
+        task.setStatus(Helper.statusName(status));
```

Comment:
Additionally, if you're calling a helper method, it's recommended that you import the static method instead of calling it using the helper class name. This means that instead of using `Helper.methodName()`, you can simply use `methodName()` directly.